.github/workflows/release.yml

```diff
@@ -61,11 +61,13 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.REPO_ACCESS_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
-      - name: Extract branch name
-        id: extract_branch
-        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+      - name: verify release required
```

Comment:
Would there be a Slack Notification for this failure?

Reply:
yes, this would come under release failure. Since we would not be doing any release